includes/RestApi/JetpackController.php

```diff
@@ -48,6 +52,19 @@ public function register_routes() {
 				),
 			)
 		);
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/regenerate_critical_css',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'regenerate_critical_css' ),
+					'permission_callback' => function () {
+						return current_user_can( 'manage_options' );
```

Comment:
We can use `permission_callback' => array( Permissions::class, 'rest_is_authorized_admin' )` instead of this.